src/result_generation/height.py

```diff
@@ -119,12 +127,53 @@ def process_depthmaps(self):
 
         return depthmaps
 
+    def process_depthmaps_MCNN(self):
```

Comment:
We should use inheritance (another class) for this workflow. 
Otherwise this class will soon get more complicated with every new workflow we introduce.
Enabling a singleartifact model should not be depedent on a multiartifact model (and the other way around)